app/src/main/java/com/duckduckgo/app/browser/BrowserTabViewModel.kt

```diff
@@ -95,7 +95,7 @@ import com.duckduckgo.app.global.model.domain
 import com.duckduckgo.app.global.model.domainMatchesUrl
 import com.duckduckgo.app.location.GeoLocationPermissions
 import com.duckduckgo.app.location.data.LocationPermissionType
-import com.duckduckgo.app.location.data.LocationPermissionsRepository
+import com.duckduckgo.app.location.data.LocationPermissionsRepositoryAPI
```

Comment:
We usually go the other way around, the implementation is the one with a different name. Also we don't capitalize the acronyms (API should really be Api).

I'd suggest changing `LocationPermissionsRepositoryAPI` should be `LocationPermissionsRepository` and `LocationPermissionsRepository` should be `LocationPermissionsRepositoryImpl` or similar